pkg/lib/config/otp.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+var _ = Schema.Add("OTPConfig", `
+{
+	"type": "object",
+	"additionalProperties": false,
+	"properties": {
+		"sms": { "$ref": "#/$defs/OTPSMSConfig" }
+	}
+}
+`)
+
+type OTPConfig struct {
+	SMS *OTPSMSConfig `json:"sms,omitempty"`
+}
+
+var _ = Schema.Add("OTPSMSConfig", `
+{
+	"type": "object",
+	"additionalProperties": false,
+	"properties": {
+		"resend_cooldown_seconds": {
+			"type": "integer",
+			"enum": [60, 120]
```

Comment:
Add test for this config. See pkg/lib/config/testdata/account_deletion_tests.yaml for an example.